app/models/olympian.rb

```diff
@@ -11,4 +11,19 @@ class Olympian < ApplicationRecord
   validates :height, presence: true
   validates :weight, presence: true
   validates :team, presence: true
+
+  # instance methods
+  def total_medals_won
+    OlympianEvent.where(olympian_id: id, medal: ["Gold","Silver","Bronze"]).count
+  end
+
+  # class methods
+  def self.youngest
+    Olympian.order(:age).take
+  end
+
+  def self.oldest
+    Olympian.order(age: :desc).take
```

Comment:
I was also thinking you could probably do .first and .last on the same method - multiple approaches with the same output.